docs/work/coverage-notes/README.md

```diff
@@ -0,0 +1,25 @@
+ 
+ # Introduction
+ When done correctly, coverage notes allow you to take the time you need off without resulting in any headaches for you or the team. In your coverage notes you should provide 3 things: What you are currently working on, what the status of your work is, and any other important notes that might be helpful to someone picking up the task. In conjunction with coverage notes, you should also talk to you PM(s) about anything else you might have missed. Your coverage notes should then be sent to your PM or most senior Dev.
```

Comment:
1. Grammarly is saying **coverage notes you should provide** should be **coverage notes, you should provide**
1. **you PM(s)** should be **your PM(s)**